test/support_bundle/persistent_volumes/pv-claim.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Move these files into the main folder only to simplify. 
Also write a simple deploy.sh or such which calls these yaml files

Also use .yml as convention for consistency

Reply:
Done.
Created a deploy_pvc.sh script will will create PV & PVCs
Renamed the deployment file to .yml